### PR TITLE
Add odf-console:release-4.12 branch

### PR DESCRIPTION
**Pull request by rexagod**
Enable CI for odf-console:release-4.12 branch.

Signed-off-by: Pranshu Srivastava <rexagod@gmail.com>

### CODE REVIEW

**Comment by vbnrh** [comment]
run 'make ci-operator-config jobs' once and commit the results

**Comment by rexagod** [comment]
It seems this is adding a particular regex on it's own.

image

**Comment by SanjalKatiyar** [comment]
> It seems that this makefile target is adding a particular regex, on it's own? This will unintentionally target the compat branch as well.
> 
> image

@vbnrh is that alright ??

**Comment by vbnrh** [comment]
@SanjalKatiyar check the rehearse. Its only running for 4.12, not the compat branch

**Comment by vbnrh** [comment]
You've only created config for 4.12

File: ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-release-4.12.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        name: odf-e2e-credentials
+        namespace: test-credentials
+      dependencies:
+      - env: ODF_CONSOLE_FULL_IMG_NAME
+        name: odf-console-img
+      from: src
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (bipuladh) [inline ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-release-4.12.yaml:74]: Why do we have such high resource requirement for running the cypress test?

Comment (bipuladh) [reply ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-release-4.12.yaml:74]: lets revert them.

Comment (bipuladh) [reply ci-operator/config/red-hat-storage/odf-console/red-hat-storage-odf-console-release-4.12.yaml:74]: See https://github.com/openshift/release/pull/32612#pullrequestreview-1132449551

**Pull request merged**